### ISSUE TITLE
make build fails on Python 3.4 with latest pkg level due to Pygments version conflict

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#521]
See here: https://travis-ci.org/github/pywbem/pywbemtools/jobs/662559173?utm_medium=notification&utm_source=github_status

Error message of make build surfacing the issue:
```
pkg_resources.VersionConflict: (Pygments 2.3.1 (/home/travis/virtualenv/python3.4.8/lib/python3.4/site-packages), Requirement.parse('Pygments>=2.5.1'))
Makefile:578: recipe for target 'dist/pywbemtools-0.5.1.dev57-py2.py3-none-any.whl' failed
```

Earlier error message during `make develop`:
```
ERROR: readme-renderer 25.0 has requirement Pygments>=2.5.1, but you'll have pygments 2.3.1 which is incompatible.
```

Dependencies:
* Collecting Pygments>=2.0 (from Sphinx>=1.7.6->-r dev-requirements.txt (line 37))
* Collecting readme-renderer>=21.0 (from twine>=1.8.1->-r dev-requirements.txt (line 53))

**Comment by andy-maier** [comment]
Analysis of the issue:

Pygments 2.4.0 (used by Sphinx) and readme-renderer 25.0 (used by twine)
have removed support for Python 3.4. The Pygments change happened already
in 5/2019 and was covered by Pip's ability to take the Python version
into account when installing the latest version of a package. However,
this ability does not cover upgrading already installed packages, and
so the issue surfaced when the installation of twine required readme-renderer
25.0 which required Pygments>=2.5.1 and along with Sphinx, Pygments 2.3.1
had already been installed.

**Comment by andy-maier** [comment]
Addressed by PR #522 for master and TBD for 0.5.1
